ldap/servers/slapd/pblock.c

```diff
@@ -4571,3 +6729,3949 @@ bind_credentials_set_nolock(Connection *conn, char *authtype, char *normdn, char
+    NULL, /* slot 1877 available */
+    NULL, /* slot 1878 available */
+    NULL, /* slot 1879 available */
+    NULL, /* slot 1880 available */
+    NULL, /* slot 1881 available */
+    NULL, /* slot 1882 available */
+    NULL, /* slot 1883 available */
+    NULL, /* slot 1884 available */
+    NULL, /* slot 1885 available */
+    NULL, /* slot 1886 available */
+    NULL, /* slot 1887 available */
+    NULL, /* slot 1888 available */
+    NULL, /* slot 1889 available */
+    NULL, /* slot 1890 available */
+    NULL, /* slot 1891 available */
+    NULL, /* slot 1892 available */
+    NULL, /* slot 1893 available */
+    NULL, /* slot 1894 available */
+    NULL, /* slot 1895 available */
+    NULL, /* slot 1896 available */
+    NULL, /* slot 1897 available */
+    NULL, /* slot 1898 available */
+    NULL, /* slot 1899 available */
+    NULL, /* slot 1900 available */
+    slapi_pblock_set_txn_ruv_mods_fn,
+    NULL, /* slot 1902 available */
+    NULL, /* slot 1903 available */
+    NULL, /* slot 1904 available */
+    NULL, /* slot 1905 available */
+    NULL, /* slot 1906 available */
+    NULL, /* slot 1907 available */
+    NULL, /* slot 1908 available */
+    NULL, /* slot 1909 available */
+    NULL, /* slot 1910 available */
+    NULL, /* slot 1911 available */
+    NULL, /* slot 1912 available */
+    NULL, /* slot 1913 available */
+    NULL, /* slot 1914 available */
+    NULL, /* slot 1915 available */
+    NULL, /* slot 1916 available */
+    NULL, /* slot 1917 available */
+    NULL, /* slot 1918 available */
+    NULL, /* slot 1919 available */
+    NULL, /* slot 1920 available */
+    NULL, /* slot 1921 available */
+    NULL, /* slot 1922 available */
+    NULL, /* slot 1923 available */
+    NULL, /* slot 1924 available */
+    NULL, /* slot 1925 available */
+    NULL, /* slot 1926 available */
+    NULL, /* slot 1927 available */
+    NULL, /* slot 1928 available */
+    NULL, /* slot 1929 available */
+    slapi_pblock_set_search_result_set_size_estimate,
+    NULL, /* slot 1931 available */
+    NULL, /* slot 1932 available */
+    NULL, /* slot 1933 available */
+    NULL, /* slot 1934 available */
+    NULL, /* slot 1935 available */
+    NULL, /* slot 1936 available */
+    NULL, /* slot 1937 available */
+    NULL, /* slot 1938 available */
+    NULL, /* slot 1939 available */
+    NULL, /* slot 1940 available */
+    NULL, /* slot 1941 available */
+    NULL, /* slot 1942 available */
+    NULL, /* slot 1943 available */
+    slapi_pblock_set_search_result_entry_ext,
+    slapi_pblock_set_paged_results_index,
+    slapi_pblock_set_aci_target_check,
+    slapi_pblock_set_dbverify_dbdir,
+    slapi_pblock_set_plugin_ext_op_backend_fn,
+    slapi_pblock_set_paged_results_cookie,
+    slapi_pblock_set_usn_increment_for_tombstone,
+    slapi_pblock_set_memberof_deferred_task,
+    NULL, /* slot 1952 available */
+};
+
+int
+slapi_pblock_set(Slapi_PBlock *pblock, int arg, void *value)
```

Comment:
I think we still need `PR_ASSERT(NULL != pblock);` here.

Reply:
> I think we still need `PR_ASSERT(NULL != pblock);` here.

Agreed, done!